Gemfile

```diff
@@ -71,5 +71,6 @@ group :test do
   gem "webdrivers"
 end
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/224972552-685b5124-12ef-4426-87c8-094e68366b80.png)

- [ ] It would be nice if you add some reasonable Future Features such as :arrow_down: to make your readme file professional and attract the viewers eyes :+1: 

* Database will be more efficient
* UI will be more human friendly